http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -585,12 +577,12 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
           )
         case x: CloudContext.Azure => F.pure(x.value)
       }
-      _ <- relayNamespaceOpt.traverse(ns =>
-        azureRelay.deleteRelayHybridConnection(
-          ns,
-          RelayHybridConnectionName(runtime.runtimeName.asString),
-          cloudContext
-        )
+
+      // Delete hybrid connection for this VM
```

Comment:
This is a good catch, explains why the connections were leaking.

Reply:
Yup, trying to verify in a BEE that they are being deleted now